src/assets/scripts/uio.js

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright the Fluidic Eleventy copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/fluidic-11ty/raw/master/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/fluidic-11ty/raw/master/LICENSE.md.
+*/
+
+"use strict";
+
+(function ($, fluid) {
```

Comment:
I don't think you actually need the anonymous function closure. You can probably just call `fluid.uiOptions.prefsEditor` directly.

Reply:
Resolved in eb161f2.